less/common/scaffolding.less

```diff
@@ -9,7 +9,7 @@
 body {
   background: @body-bg;
   color: @text-color;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif;
+  font-family: system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Ubuntu, Cantarell, Oxygen, Roboto, Helvetica, Arial, sans-serif;
```

Comment:
Does anybody understand why GitHub seems to think that the leading dash changed somehow?

Reply:
Not seeing why. Pretty odd though 🤔

Reply:
Ah, the mysteries of life :smile: